thesheriff/infrastructure/repository/mysql_raid_repository.py

```diff
@@ -83,15 +92,15 @@ def update(self, mod_raid: Raid) -> NoReturn:
             self.__raid_table.c.id == mod_raid.id).values(**mod_raid)
         self.__connection.execute(query)
 
-    def __join_bandidos_ids(self):
-        ids = [bandido.id for bandido in self.bandidos]
+    def __join_outlaw_ids(self, outlaws: List[Outlaw]) -> str:
+        ids = [str(outlaw.id) for outlaw in outlaws]
         return ','.join(ids)
 
-    def __split_outlaws_ids(self, outlaws: str) -> List[int]:
+    def __split_outlaw_ids(self, outlaws: str) -> List[int]:
```

Comment:
Maybe in a `Formatter `object? I'm sure We have to do the same, at least, for  outlaw's friends